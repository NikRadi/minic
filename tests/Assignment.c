int main() {
    int x1;
    x1 = 3;
    printf("%d\n", x1);


    int x2 = 3,
        x3 = 5;
    printf("%d\n", x2 + x3);


    char x9 = 10;
    char x10 = x9 + 20;
    printf("%d %d\n", x9, x10);


    int x4;
    int x5;
    x4 = x5 = 3;
    printf("%d\n", x4 + x5);


    int x6 = 4;
    int x7 = 6,
        x8 = 1;
    printf("%d\n", x6 - x8 + x7);
}
