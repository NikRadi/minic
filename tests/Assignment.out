3
8
6
9